addons/web/static/src/views/form/form_controller.scss

```diff
@@ -87,6 +87,14 @@
     .oe_inline, .o_field_widget.oe_inline > * {
         width: auto!important;
     }
+    .oe_inline.o_field_float_time input {
+        width: 5ch !important;
+    }
+    @for $i from 2 through 15 {
+        .oe_inline.o_input_#{$i}ch input{
+            width: 1ch * $i !important;
+        }
+    }
```

Comment:
I saw we only need up to 12ch, let's not have three extra rules never used.

Also, do we still need to limit `to oe_inline` for this class to work or can we remove this?

Can you also briefly explain here (for reviewers) why we need the `!important`?

And could you add links to other PR on this PR message?

Thanks :-)